shell/lib/asdf.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 # Utilities for working with asdf
 
+asdf_global_plugins=("golang 1.19.5")
```

Comment:
Should we read the current go version out of the projects `.tool-versions` instead of hardcoding this here?

Reply:
I thought about that, but I'm torn -- that ties the version we expect to use to install plugins to the version that the repo is using, which feels like we should keep it dissociated.  Then you have your build system's build system (to assemble your build system), then separately your build system builds your app, and they're uncoupled.

Reply:
Talked offline and updated the method